client/src/actions/publish.js

```diff
@@ -14,6 +14,12 @@ export function clearFile () {
   };
 }
 
+export function setUpdateTrue () {
```

Comment:
Is this necessary? My inclination is that there must be another state value corresponding to what is being edited (or even however you ended up on the edit page) that should allow you to know this without an additional explicit value.

Reply:
I think it would be more complex to do this another way e.g. looking at the router path